app/javascript/components/map/components/legend/components/timeline/index.js

```diff
@@ -34,184 +24,36 @@ const mapStateToProps = (
 };
 
 class TimelineContainer extends PureComponent {
-  static defaultProps = {
-    dateFormat: 'YYYY-MM-DD',
-    interval: 'years',
-    step: 1,
-    speed: 200,
-    count: 2,
-    pushable: 0
-  };
-
-  constructor(props) {
-    super(props);
-    const { minDate, maxDate, startDate, endDate, trimEndDate } = props;
-    this.state = {
-      isPlaying: false,
-      min: 0,
-      max: dateDiffInDays(maxDate, minDate),
-      start: dateDiffInDays(startDate, minDate),
-      end: dateDiffInDays(endDate, minDate),
-      trim: dateDiffInDays(trimEndDate, minDate),
-      loops: 0
-    };
-  }
-
-  componentDidUpdate(prevProps, prevState) {
```

Comment:
I guess most of this logic is now in vizzuality-components?

Reply:
yup, nearly all of it!